apisix/plugins/proxy-mirror.lua

```diff
@@ -26,6 +26,12 @@ local schema = {
             pattern = [[^http(s)?:\/\/[a-zA-Z0-9][-a-zA-Z0-9]{0,62}]]
                       .. [[(\.[a-zA-Z0-9][-a-zA-Z0-9]{0,62})+(:[0-9]{1,5})?$]],
         },
+        percentage = {
```

Comment:
IMHO, we use sample_ratio for this purpose in skywalking & zipkin. What about using this name?

Reply:
okay~